### DISCUSSION TITLE
How to start using AWS Cpp-SDK for Minio

### DISCUSSION THREAD

**Discussion by Baharak12** [minio/minio#12622] [Q&A]
As part of my work with cloud S3 buckets, I have already used the AWS-SDK for C++. In MINIO local server, I would like to upload some objects to S3 buckets using AWS SDK for C++ library.
Could someone please tell me how I can use the AWS-SDK CPP library in Linux to upload some files to MINIO's s3 bucket?

How can I use the library to start the connection with Minio using the entire AWS SDK C++ examples repository?

**Comment by harshavardhana** [comment]
Moving this to discussion - this is not an issue..
